recipes/datasette/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "datasette" %}
+{% set version = "0.26.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3856bf31482428dd89e965ebd090b69a844bac814f807e3317c05a0c631dfb57
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - click
+    - click-default-group
+    - sanic
+    - jinja2
+    - hupper
+    - pint
+    - pluggy
+
+test:
+  commands:
+    - datasette --help
```

Comment:
You need to specify an entry point for this to work.

Reading the project's [`setup.py`](https://github.com/simonw/datasette/blob/1f91065b20cbc691f464bccfd8eef7d1ce4b14a8/setup.py#L47)  I suggest you add the following to the `build:` section:
```
  entry_points:
            - datasette = datasette.cli:cli
```